pkg/jsonstream/format.go

```diff
@@ -23,46 +22,21 @@ var (
 	jsonSep        = []byte(",")
```

Comment:
`jsonBeginDelim`, `jsonEndDelim` and `jsonSep` seems have been deprecated?

And the `Formater` seems not appropriate as well 😄 . Only the `Write` method of it is necessary.

Reply:
I will remove the `jsonBeginDelim`, `jsonEndDelim`, `jsonSep`. Maybe the requirement is to render the array json to the caller. But for this case, we don't need the `BeginWrite` function.